artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/PostOfficeImpl.java

```diff
@@ -1211,6 +1211,7 @@ public MessageReference reload(final Message message, final Queue queue, final T
          }
       }
 
+      queue.refUp(reference);
```

Comment:
This causes org.apache.activemq.artemis.tests.integration.client.LargeMessageTest#testFilePersistenceBlockedPreACK to fail. 
the large message file does not get removed.
Without it, it seems to work but the ref count is -1 on delete, which seems wrong. it may be an ordering issue

Reply:
did you see my comment.. I sent a fix on another PR.